nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss feature mediabuttons 13

# Description - Created Capstone & Podcast buttons in DevCard

## Type of change - added buttons


# Testing Instructions
Fetch repo and checkout into ss-feature-mediabuttons-13 branch and see if buttons show up on DevCard

ADDITION:
Added Capstone button
Added Podcast player 

# Checklist:
